mix.exs

```diff
@@ -23,7 +23,8 @@ defmodule Phoenix.Mixfile do
      env: [code_reloader: false,
            template_engines: [],
            format_encoders: [],
-           pubsub: [garbage_collect_after_ms: 60_000..300_000],
+           pubsub: [adapter: Phoenix.PubSub.PG2Adapter,
```

Comment:
Should all be configurations in the supervision tree when starting. One option is to start the adapter directly, instead of a proxy thing.